build_riscv.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -x
+
+if [ -z $PLATFORM ]; then
+	echo "Set a platform"
+	exit 1
+fi
+
+# This is currently just hifive1
+if [ $PLATFORM == "riscv32" ]; then
+	TARGET=riscv32imac-unknown-none-elf
+	ELF_FILE=rv32imac.elf
+else
+	TARGET=riscv32imc-unknown-none-elf
+	ELF_FILE=rv32imc.elf
+fi
+
+cargo build --release --target=$TARGET --example $1
+
+elf_file_name="target/tab/$1/$ELF_FILE"
+tab_file_name="target/tab/$1.tab"
+
+mkdir -p "target/tab/$1"
+cp "target/$TARGET/release/examples/$1" "$elf_file_name"
+
+elf2tab --verbose -n "$1" -o "$tab_file_name" "$elf_file_name" --stack 2048 --app-heap 1024 --kernel-heap 1024 --protected-region-size 64
```

Comment:
This should be moved to flash.sh

Reply:
Yes. Even if `tockloader` isn't or can't be used at the moment `flash.sh` can act as a generator for the `tab` file.

Reply:
Can we not use `-n` it doesn't work on the latest `elf2tab`

Reply:
the problem is that 7b347b74e44141f9effc691eb371d07a1dc5566e changed the alias from -n to -p and older versions don't actually have the --package-name option. This should probably be fixed in elf2tab.